### PR TITLE
feat: classified domains calling context (AR-1557)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1557" title="AR-1557" target="_blank">StoryAR-1557</a>  Federated status of remote call participant
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Creates a component to display conversation/call classification banner and applied into calling context, incoming and ongoing calls.

#### How to Test

You can create an account into `ANTA` or `BELLA` and do some calls between groups or one to one call across backends.
You should see the banner as `Classified` accordingly if they are supported as feature flag and participants are trusted.

### Notes (Optional)

More context about classification of conversations and backends can be found here:

https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/376439791/Use+case+determine+if+a+conversation+or+user+is+classified+non-classified

### Attachments (Optional)

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+@Preview(showBackground = true)
+@Composable
+fun ClassifiedIndicatorPreview() {
+    Column(modifier = Modifier.fillMaxWidth()) {
+        ClassifiedIndicatorBanner(classifiedType = ClassifiedType.CLASSIFIED)
+
+        Divider()
+
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt:81]:
```suggestion
        Divider()
```

File: app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.wire.android.R
+import com.wire.android.ui.theme.wireTypography
+import com.wire.kalium.logic.feature.conversation.ClassifiedType
+
+@Composable
+fun ClassifiedIndicatorBanner(
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt:23]: It reads a bit odd to me. Maybe `ClassificationLevelBanner` or `SecurityClassificationBanner` would be more self explanatory?

**Review by gongracr — APPROVED** [review]
Looking spotless as usual compa ✨ 
The only thought I have is about naming (once again 😅 ) as `classifiedXxx` might read a bit confusing for someone w/o any context.
Let me know what you think. I will approve it in any case 🦞

File: app/src/main/kotlin/com/wire/android/ui/calling/CallState.kt [hunk 3]
```diff
@@ -16,4 +17,5 @@ data class CallState(
     val isCameraFlipped: Boolean = false,
     val conversationType: ConversationType = ConversationType.OneOnOne,
     val membership: Membership = Membership.None,
+    val classifiedType: ClassifiedType = ClassifiedType.NONE,
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/calling/CallState.kt:20]: Same here, maybe `securityClassificationType` ?

File: app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt (hunk 2 above)
Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt]: Applied ! :D

**Review by alexandreferris — APPROVED** [review]
lookin' neat 👌🏻

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-916/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4299353)(4299353)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4299353)(4299353)-dev-release.apk)

**Pull request merged**